.pre-commit-config.yaml

```
@@ -0,0 +1,22 @@
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.8.2
+    hooks:
+      - id: ruff
+        args: [--fix]
+      - id: ruff-format
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)